### ISSUE TITLE
Can I use ogg/vorbis file as training data?

### ISSUE THREAD

**Issue by mychiux413** [pytorch/fairseq#2729]
#### What is your question?
Hi,
I want to finetune the wav2vec2.0 CTC model from base model,
and I know the `soundfile` can read the `ogg/vorbis` file now,
but the array values read from `.ogg` files could be **GREATER THAN 1.0**, **IF** we use `soundfile` to read it (see [libsndfile issue](https://github.com/libsndfile/libsndfile/issues/16#issuecomment-7751265) )

so, the question is:
* Will it affect the accuracy of the model, if I train the model with `.ogg` files (or even infer `.ogg` with pretrained model)? If so, maybe we should warn developers to prevent the use of `.ogg` files

P.S. I want to use `.ogg` because of my disk space is too small, so I have to compress those wavs into ogg

#### Code

To prove that the value from `.ogg` can be greater than 1.0

``` py
import soundfile
# prepare a temp.wav file first
arr, sr = soundfile.read('temp.wav')
print(arr.min(), arr.max())
# -0.31622776621952653 0.31622776575386524

# maxmize signal to 1.0
arr /= max(abs(arr.min()), arr.max())
print(arr.min(), arr.max())
# -1.0 0.9999999985274497

# save as .ogg
soundfile.write('temp.ogg', arr, format='ogg', samplerate=sr)

# read it back
arr, sr = soundfile.read('temp.ogg')
print(arr.min(), arr.max())
# -1.2072484493255615 1.1288540363311768

```

#### What have you tried?
I also tried `pydub` to read `.ogg`, and it can compress the signal values into: -1.0~1.0, but the reading speed is very slow (10X than `soundfile`)

#### What's your environment?

 - soundfile version: 0.10.2

**Comment by kahne** [comment]
@mychiux413 Did you try encoding OGG with other libraries (e.g. torchaudio which has SoX backend on Linux/OSX) since this is likely a bug in soundfile (libsndfile)?

**Comment by mychiux413** [comment]
@kahne No, I only tried `pydub` (libvorbis?), and everything was fine.

BTW, I also tried to compress everything includes  into `.ogg` and trained `wav2vec2` model,
and the accuracy was still great.
